src/Sarif.Viewer.VisualStudio/Fixes/EditActionPreviewProvider.cs

```diff
@@ -134,7 +137,14 @@ public class EditActionPreviewProvider : IPreviewProvider
             var sizeFitter = new SizeToFitHelper(diffViewer, 400.0);
             await sizeFitter.SizeToFitAsync();
 
-            return new DisposableDifferenceViewerControl(diffViewer, description, additionalContent);
+            var diffViewerControl = new DisposableDifferenceViewerControl(errorListItem, diffViewer, description, additionalContent);
+            diffViewerControl.ApplyFixesInDocument += this.Control_ApplyFixesInDocument;
```

Comment:
>+= [](http://example.com/codeflow?start=51&length=2)

is it possible to unhook this?

Reply:
seems the wpf control lifecycle is not controlled by code. 
but since the reference is from control object (publisher) to provider (subscriber), it should be recycled when control object is disposed.

Reply:
got it. added a comment in the disposabledifference class